docs/stencil-docs/reference-docs/front-matter-reference.md

```diff
@@ -255,12 +261,44 @@ gql: "query productById($productId: Int!) {
   }
 }"
 ```
-  
+
+<!--
+type: tab
+title: Category
+-->
+
+ ```yaml title="Front matter block in category.html template" lineNumbers
+category:
+    shop_by_price: true
+    products:
+        limit: {{theme_settings.categorypage_products_per_page}}
+gql: "query categoryById ($categoryId: Int!){
+	site {
+    category(entityId: $categoryId) {
+      products {
+        edges {
+          node {
+            sku
+          }
+        }
+      }
+    }
+  }
+}"
+```
+
+<!-- type: tab-end -->
+
 We suggest testing GraphQL queries using the [storefront API playground](https://developer.bigcommerce.com/graphql-playground) to refine them before adding them to your template. You can launch the playground in the context of your store by clicking the **Storefront API Playground** link under the **Advanced Settings** menu in your store's control panel.
   
-Once you have added a query to your template's front matter block, execution happens automatically when the page loads. The data returned by the query will be returned in the page's context and made available to the handlebars under the `gql` key. For example, you can retrieve the variant data from the above query in `product.html` like this:
+Once you have added a query to your template's front matter block, execution happens automatically when the page loads. The data returned by the query will be returned in the page's context and made available to the handlebars under the `gql` key. The following examples illustrate how you can retrieve the variant data from the above query in `product.html`, as well as a category's product data from the above query in `category.html`:
 
-```handlebars lineNumbers
+<!--
+type: tab
+title: Product
+-->
+
+```handlebars title="Handlebar in product.html template" lineNumbers

```

Comment:
```suggestion
```handlebars title="Example gql Handlebars object in product.html template" lineNumbers

```